pgctl/cli.py

```diff
@@ -20,16 +21,28 @@
 }
 
 
+def close_fds():
+    import os
```

Comment:
Probably worth a comment:

# The builtin Popen close_fds doesn't close stdout, stderr, but we must in order to daemonize properly.